apps/explorer/src/ui/TransactionType.tsx

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { type TransactionKindName } from '@mysten/sui.js';
+
+import { ReactComponent as ContentSuccessStatus } from './icons/check_12x12.svg';
+import { ReactComponent as ContentFailedStatus } from './icons/x.svg';
+
+type TransactionTypeLabel = TransactionKindName | 'Batch';
+
+export type TransactionTypeProps = {
+    isSuccess?: boolean;
+    count?: string;
+    type?: TransactionTypeLabel;
+};
+
+export function TransactionType({
+    isSuccess,
+    count,
+    type,
+}: TransactionTypeProps) {
+    return (
+        <div className="flex flex-col items-start">
+            <div className="flex items-center justify-center gap-1.5">
```

Comment:
I think if you remove `justify-center` from here you can get rid the other div

Reply:
Excellent - I tried this and it looks like this works as a refactoring.